docs/source/pages/DM9098.rst

```diff
@@ -0,0 +1,126 @@
+more information below at *Getting started*.
+
+How it works
+************
+
+OAK-D-Pro has **notch IR filters** (940nm) on mono cameras, which allows only visible light and IR light from illumination LED/laser dot
+projector to the camera.
+
+**Laser dot projector** projects many small dots in front of the device, which helps with disparity matching, especially for low-visual-interest surfaces
+(blank surfaces with little to no texture), such as a wall or floor. Technique that we use is called `ASV <https://en.wikipedia.org/wiki/Computer_stereo_vision#Conventional_active_stereo_vision_(ASV)>`__
+- conventional active stereo vision - as stereo matching is performed on device the same way as on OAK-D (passive stereo).
+
+**Blanket IR LED** illumination allows perceiving low-light and no-light environments. You can run your AI/CV processes
+on frames that are illuminated by the IR LED. Note that color camera doesn't percieve IR light, so you would need to use mono camera stream
+for your AI/CV processes.
+
+Getting started
+***************
+
+To get started with OAK-D-Pro, you first need to install the correct version: checkout to branch :code:`oak-d-pro` in `depthai <https://github.com/luxonis/depthai>`__
+and execute :code:`python3 install_requirements.py`.
+This will install the correct library, so you can execute :code:`depthai_demo.py`. Control panel for configuring IR LED/laser projector should appear,
+as shown on screenshots below.
+
+Sensor specifications
+*********************
+
+.. list-table::
+   :widths: 30 30 30
+   :header-rows: 1
+
+   * - Camera Specs
+     - Color camera
+     - Stereo pair
+   * - Sensor
+     - IMX378
+     - OV9282
+   * - DFOV
+     - 81°
+     - 81°
+   * - HFOV
+     - 69°
+     - 72°
+   * - Resolution
+     - 12MP (4032x3040)
+     - 1MP (1280x800)
+   * - Focus
+     - Auto-Focus: 8cm - ∞
+     - Fixed-Focus: 19.6cm - ∞
+   * - Max Framerate
+     - 60 FPS
+     - 120 FPS
+   * - F-number
+     - 2.0
+     - 2.2
+   * - Lens size
+     - 1/2.3 inch
+     - 1/2.3 inch
+
+Projector specifications
+************************
+
+.. list-table::
+   :widths: 1 1
+   :header-rows: 1
+
+   * - Projector Specs
+     - Value
+   * - Dot projector
+     - Ams Belago1.1 Dot-Pattern Infrared Illuminator
+   * - Number of dots
+     - 4700
+   * - HFOI* 50%
+     - 78 ± 7%
+   * - VFOI* 50%
+     - 61° ± 7%
+   * - VSCEL wavelength
+     - 940nm
+   * - Operating temperature
+     - 0°C to ~80°C
+   * - PDF
+     - `Click here <https://ams.com/documents/20143/4410698/Belago1.1_DS000676_1-00.pdf>`__
+
+*FOI = Field of illumination*. Also note that in datasheet, HFOI and VFOI are switched, that's because we mount the Belago1.1 rotated as we want
+greater horizontal field, to match field of cameras.
+
+Demo
+****
+
+As you can see from :code:`color` frame on screenshots, it's pitch dark in the room, so the only light source is the IR illumination LED/dot projector.
```

Comment:
would be good to add that device was not factory calibrated but calibrated manually by hand (depth performance is worse because of that)

Reply:
@GNejc I believe there was something about adding laser safety content to any marketing/guides/datasheets docs.
I meant we should add what is needed to the web page @Erol444 is working on.

Reply:
Ahh, I thought you meant only on the device. Yes for the docs, we have this at the moment. It will probably change in the future as this is only a placeholder for now.
![laser-safety warning](https://user-images.githubusercontent.com/88615832/146359167-7acb9b67-1e73-4b06-a694-81b9002c2c38.png)
.

Reply:
For documents this is how I put it in the user manual for the pro as a reference (under "certification statement").
https://docs.google.com/document/d/1XUNk7qKfflq19fTJ7VvdB0HjN14te2Vg/edit#heading=h.1ksv4uv